changelog/24297.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
```

Comment:
I'm not sure how we should categorize this, it's a bug, but the impact is that files go to the right place instead of the timestamp suffixed one, which might catch some users out. :(